en/guide/sdk.md

```diff
@@ -0,0 +1,155 @@
+# DroneCore SDKs
+
+DroneCore allows SDK Vendors to create independent plugin libraries, along with integration and unit tests, and 
+build them into DroneCore at compile time. <!-- It also provides other functionality that is useful for SDK developers,
+including the ability to lock down custom versions of DroneCore to authorised developers. -->
+
+> **Note** This topic only covers the (very minor) differences between SDK plugins/testing and "vanilla" DroneCore.
+> SDK authors should review the topics [Writing Plugins](../contributing/plugins.md) and 
+> [Testing](../contributing/test.md).
+
+
+## SDK Architecture
+
+SDK extensions are defined in a separate/parallel folder to DroneCore. 
+The extension folder *mirrors* DroneCore, and **must** contain the folders **integration_tests** and **plugins**.
+The *plugin* folder contains appropriately named sub-folders for each plugin (along with its unit test code).
+
+A simplified view of a "typical" DroneCore SDK is shown below (with extension directory: *SDK_Extensions*). 

```

Comment:
I would say:

DroneCore can be extended with plugins and integration tests that are defined out of tree. This way they can be checked in to a separate repository which reduces git conflicts.